pkg/i18n/dutch.go

```diff
@@ -82,12 +82,18 @@ func addDutch(i18nObject *i18n.Bundle) error {
 		}, &i18n.Message{
 			ID:    "refresh",
 			Other: "verversen",
+		}, &i18n.Message{
+			ID:    "push",
+			Other: "push",
+		}, &i18n.Message{
+			ID:    "pull",
+			Other: "pull",
 		}, &i18n.Message{
 			ID:    "addPatch",
 			Other: "verandering toevoegen",
 		}, &i18n.Message{
 			ID:    "edit",
-			Other: "verander",
```

Comment:
I think **bewerken** is a better translation

Reply:
agreed